README.md

```diff
@@ -2,24 +2,197 @@
- * **map style** that highlights CERN buildings, following the [Mapbox Style Specification](https://www.mapbox.com/mapbox-gl-js/style-spec/);
- * **Dockerfile** which prepares a container that is capable of serving the tiles, based on klokantech's [tileserver-gl](https://github.com/klokantech/tileserver-gl);
 
-The Dockerfile is made up of two stages:
- * the first one downloads the [OpenStreetMap data](https://planet.osm.ch/), crops it to a bounding box and generates an `*.mbtiles` file containing relevant data;
- * the second one sets up a lightweight tileserver-gl using the data generated above;
+## Introduction
 
-It should be quite easy to fork this repo and adapt it to your own context. Shape files were generated using [QGIS](https://qgis.org), from OSM data.
+The Indico Tile Server is based on the [tileserver](https://github.com/klokantech/tileserver-gl) application. Tiles are
+the map pieces that are served to the client through HTTP. They are store in a file
+that can be built with the [tilemaker](ttps://github.com/systemed/tilemaker/blob/master/CONFIGURATION.md) application.
 
-This was possible thanks to the [OpenStreetMap project](https://www.openstreetmap.org/).
+`tilemaker` is using [OpenStreetMap](https://www.openstreetmap.org) (OSM) maps as input and
+applies some rendering instructions (style) to the input map, using the [Lua](https://www.lua.org)
+scripting language. The rendering definition can be based on shapes, typically polygons, that
+define areas of the map that must be rendered specifically (for example to highlight the
+buildings of your laboratory in a campus). Shapes must be built with a tool like
+[QGIS](https://www.qgis.org) and stores in a [shapefile](https://wiki.openstreetmap.org/wiki/Shapefiles).
+
+Even though it is not mandatory, the tile server as provided in this repository is intended to 
+be run as a Docker container.  For this reason, this repository provides, in
+ addition to configuration files for `tileserver` and `tilemaker`, a Dcokerfile to build
+the container.
+All these files are based on CERN configuration and can be used as templates to build your own
+configuration. The documentation below describes the main parameters that need to be customized.
+
+## Building the tileserver container
+
+### Dockerfile
+
+The Dockerfile is made up of two stages: each stage produces a different container.
+
+* the first one downloads the [OpenStreetMap data](https://planet.osm.ch/), crops it to a bounding box and generates an 
+`*.mbtiles` file containing relevant data;
+  * Relies on [osmconvert](https://wiki.openstreetmap.org/wiki/Osmconvert) tool to extract the region
+of interest from an OSM map.
+* the second one sets up a lightweight tileserver-gl using the data generated above;
+
+The Dockerfile must be customized to use the proper GSM coordinates and the configuration files you will create in
```

Comment:
Do you mean "GIS coordinates"?

Reply:
In fact if was GPS... thanks for spotting this mistake!